docs/self-managed/optimize-deployment/configuration/getting-started.md

```diff
@@ -0,0 +1,123 @@
+
+## Camunda Platform 8 specific configuration
+
+For Camunda Platform 8, Optimize is importing process data from exported zeebe records as created by the [Zeebe Elasticsearch Exporter](https://github.com/camunda/zeebe/tree/main/exporters/elasticsearch-exporter) from the same Elasticsearch cluster that Optimize used to store it's own data. For the relevant configuration options, refer to the [Camunda Platform 8 import configuration](./system-configuration-platform-8.md).
+
+## Recommended additional configurations
+
+### Adjust engine heap size
+
+Sending huge process definition diagrams via Rest API might cause the engine to crash if the engine heap size is inadequately limited. Thus, it is recommended to increase the heap size of the engine to at least 2 GB; for example, by adding the following Java command line property when starting the engine:
+
+```bash
+-Xmx2048m
+```
+
+Also, it is recommended to decrease the [deployment cache size](https://docs.camunda.org/manual/latest/user-guide/process-engine/deployment-cache/#customize-the-maximum-capacity-of-the-cache) to `500`, e.g. by:
+
+```bash
+<property name="cacheCapacity" value="500" />
+```
+
+### Adjust Optimize heap size
+
+By default, Optimize is configured with 1GB JVM heap memory. Depending on your setup and actual data, you might still encounter situations where you need more than this default for a seamless operation of Optimize. To increase the maximum heap size, you can set the environment variable `OPTIMIZE_JAVA_OPTS` and provide the desired JVM system properties; for example, for 2GB of Heap:
+
+```bash
+OPTIMIZE_JAVA_OPTS=-Xmx2048m
+```
+
+### Maximum result limits for queries
+
+It's possible that engine queries [consume a lot of memory](https://docs.camunda.org/manual/latest/user-guide/process-engine/process-engine-api/#query-maximum-results-limit). To mitigate this risk, you can [limit the number of results](https://docs.camunda.org/manual/latest/reference/deployment-descriptors/tags/process-engine/#queryMaxResultsLimit) a query can return. If you do this, it is highly recommended that you set the value of the `queryMaxResultsLimit` setting to `10000` so that the Optimize import works without any problems. This value should still be low enough so you don't run into any problems with the previously mentioned heap configurations.
+
+## Hardware resources
+
+We recommend to carefully choose hardware resources that are allocated to the server with Optimize.
+
+Be aware that Optimize is using data structures that are different from data stored
+by the Camunda Platform 7 engine. The final amount of space on the hard drive required by Optimize will
+depend on your replication settings, but as a rule of thumb, you could expect Optimize to use 30% of the space that
+your relational database is using.
```

Comment:
Should this be generic for "Camunda Platform"? So we can cover both 7 and 8? Or is this specific to 7? (This is likely a @megglos question.)

Reply:
It's actually specific to C7. I would actually go for removing the whole section here. It starts with recommending choosing the Optimize server hardware carefully and then speaks about actually the elasticsearch database this is misleading as this should be separate servers anyway and it doesn't add much value but rather confusion.

Reply:
Removed this portion in latest commit.